doc/walkthrough.rst

```diff
@@ -0,0 +1,139 @@
+Walkthrough
+===========
+
+This document walkthroughs a simple scenario with Commissaire.
+
+Configuring a ContainerManager (Optional)
+-----------------------------------------
+If you will be using OpenShift, OCP, or Kubernetes then configuring a ContainerManager is the first thing to do.
+This essentially will tell commissaire how to communicate with your ContainerManager. When a cluster is associated
+to this ContainerManager new hosts will be automatically added into the the ContainerManager as nodes.
+
+Let's say you wanted to add a ContainerManager called ``ocp``,  which has a url of ``https://openshift.example.com``,
+and uses a token of ``aaa`` for authentication:
+
+.. include:: examples/commctl_create_container_manager.rst
+
+.. note::
+
+    "Adding Hosts To The Cluster" later in this document will show how the ContainerManager
+    interacts binds with Clusters and Hosts.
```

Comment:
Also :+1: to adding an xref. I think the RST change would be to add a section label above `Adding Hosts To The Cluster`

```rest
.. _walkthrough_adding_hosts_to_the_cluster

Adding Hosts To The Cluster
---------------------------
```

and link it in like

```rest
.. note::

   :ref:`Adding Hosts To The Cluster <walkthrough_adding_hosts_to_the_cluster>`, later in this document, will show how...

```